### DISCUSSION TITLE
Update Notification Feature Poll

### DISCUSSION THREAD

**Discussion by Jamminroot** [Jamminroot/intag#17] [Polls]
I've been considering the addition of a feature that allows checking for updates and displaying a notification if there's a newer version available. However, this feature would introduce new code for web requests and may impact the goal of keeping the tool as small and efficient as possible. On the other hand, a notification via email list may also be a solution, but I personally do not use email lists for this purpose.

I wanted to hear your thoughts and get your feedback on this decision. 

This poll is going to be open for a while, giving all users ample time to consider their options and make their voices heard. Whether you're a frequent or seldom user, your input is valuable and will help shape the tool.

**Comment by NextDev65** [comment]
Sign into github, then on the repo page (top right) `Watch > Custom >` and tick only `Releases`
You should get **email notifs** every release now, and if not you can set that in github settings.

**Reply by Jamminroot** [reply]
Fair use for notifications on GH, but afaik some users don't download intag from github (no idea where from; Also I do realise that this poll in that scenario is kind of one-sided :D ), and some don't feel like bothering with registration here, unfortunatelly.
Otherwise that should solve the problem for ppl here.

**Comment by Oxika95** [comment]
I think another option of just having a [Check for updates] link would be sufficient. just a hyperlink to the github releases page.

**Comment by Jamminroot** [comment]
Handled by the store; Might consider manual check for updates for folks who decided to go with the GH release installation
